components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -0,0 +1,257 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomNavigationBar.h"
+
+#import "MaterialMath.h"
+#import "MaterialRTL.h"
+#import "private/MDCBottomNavigationItemView.h"
+
+static const CGFloat kMDCBottomNavigationBarHeight = 72.f;
```

Comment:
How does this work on an iPhone X?

Reply:
This has been updated to account for the iPhone X bottom inset when necessary.

![simulator screen shot - iphone x - 2017-10-10 at 10 07 35](https://user-images.githubusercontent.com/760941/31391482-aa19dfe4-ada4-11e7-8417-4197cf33e599.png)
![simulator screen shot - iphone x - 2017-10-10 at 10 07 40](https://user-images.githubusercontent.com/760941/31391483-aa280074-ada4-11e7-94a6-d2d418aeea72.png)